resources/commands.js

```diff
@@ -877,25 +877,24 @@ status.command({
         name: "amount",
         type: status.types.NUMBER
     }],
-    preview: function (params) {
-        return status.components.text(
-            {},
-            params.amount + " ETH"
-        );
-    },
     handler: function (params) {
         return {
             event: "request",
             params: [params.amount],
             request: {
                 command: "send",
+                contentCommand: "request",
```

Comment:
Instead of specifying this information here we could get it from parameters here https://github.com/status-im/status-react/blob/8663bdfada6f1d7690a6fb0ab1a88236eaef0e34/src/status_im/chat/handlers/send_message.cljs#L286-L286
Command's name can be found in the same map as `chat-id` and `command`, the path is `[:command-name :command :name]`.
It would be great to start using specs there, but that's not objective of this PR.